browser/fragments/ably-epilogue.js

```diff
@@ -1,16 +1,35 @@
-if(typeof Realtime !== 'undefined') {
-	Ably.msgpack = msgpack;
-	Ably.Rest = Rest;
-	Ably.Realtime = Realtime;
-	Realtime.ConnectionManager = ConnectionManager;
-	Realtime.BufferUtils = Rest.BufferUtils = BufferUtils;
-	if(typeof(Crypto) !== 'undefined') Realtime.Crypto = Rest.Crypto = Crypto;
-	Realtime.Defaults = Rest.Defaults = Defaults;
-	Realtime.Http = Rest.Http = Http;
-	Realtime.Utils = Rest.Utils = Utils;
-	Realtime.Http = Rest.Http = Http;
-	Realtime.Message = Rest.Message = Message;
-	Realtime.PresenceMessage = Rest.PresenceMessage = PresenceMessage;
-	Realtime.ProtocolMessage = Rest.ProtocolMessage = ProtocolMessage;
-}
+	if(typeof Realtime !== 'undefined') {
+		Ably.msgpack = msgpack;
+		Ably.Rest = Rest;
+		Ably.Realtime = Realtime;
+		Realtime.ConnectionManager = ConnectionManager;
+		Realtime.BufferUtils = Rest.BufferUtils = BufferUtils;
+		if(typeof(Crypto) !== 'undefined') Realtime.Crypto = Rest.Crypto = Crypto;
+		Realtime.Defaults = Rest.Defaults = Defaults;
+		Realtime.Http = Rest.Http = Http;
+		Realtime.Utils = Rest.Utils = Utils;
+		Realtime.Http = Rest.Http = Http;
+		Realtime.Message = Rest.Message = Message;
+		Realtime.PresenceMessage = Rest.PresenceMessage = PresenceMessage;
+		Realtime.ProtocolMessage = Rest.ProtocolMessage = ProtocolMessage;
+	}
 }).call({});
+
+/* CommonJS support */
+if (typeof exports === "object" && exports) {
```

Comment:
...Wow. [Right you are](https://tc39.github.io/ecma262/#sec-typeof-operator). I did not know that. That an... _interesting_ design choice.